lib/internal/Magento/Framework/Data/Collection.php

```diff
@@ -500,16 +500,19 @@ public function clear()
      */
     public function walk($callback, array $args = [])
     {
-        $results = [];
+    	 $results = [];
         $useItemCallback = is_string($callback) && strpos($callback, '::') === false;
         foreach ($this->getItems() as $id => $item) {
+            $params = $args;
             if ($useItemCallback) {
                 $cb = [$item, $callback];
             } else {
                 $cb = $callback;
-                array_unshift($args, $item);
+                array_unshift($params, $item);
             }
-            $results[$id] = call_user_func_array($cb, $args);
+            //@codingStandardsIgnoreStart
+            $results[$id] = call_user_func_array($cb, $params);
+            //@codingStandardsIgnoreEnd
```

Comment:
Nothing, I was just getting a a warning that you should not use "call_user_func_array". I just removed this coding standard ignores